std/digest/digest.d

```diff
@@ -383,7 +384,26 @@ unittest
     myFunction!CRC32();
 }
 
-private template isDigestibleRange(Range)
+/**
+ * Checks whether the digest has a $(D blockSize) member, which contains the
+ * digest's internal block size in bits. It is primarily used by $(XREF digest.hmac, HMAC).
+ */
+
+template hasBlockSize(T)
+if(isDigest!T)
```

Comment:
space after `if`, `for`, `foreach` etc. throughout - this is phobos style so kinda hard to negotiate against :)